input.py

```diff
@@ -0,0 +1,32 @@
+from typing import Counter
+from unittest.result import failfast
+from spellchecker import SpellChecker
+
+
+def function(sting):
+    userList = sting.split()
+    Counter=0

```

Comment:
Is a counter some function? The capitalization confuses me, we typically name variables in lowercase (with _ between words)
```suggestion
    counter=0

```